packages/shared/src/components/sidebar/Sidebar.spec.tsx

```diff
@@ -212,3 +212,37 @@ it('should show the my feed items if the user has filters', async () => {
   const section = await screen.findByText('My feed');
   expect(section).toBeInTheDocument();
 });
+
+it('should show the migrated message if the user has existing filters', async () => {
```

Comment:
I'm missing a test to check that `updateAlerts` is called

Reply:
That is the first test on line 118.
Felt a bit overkill to test it everytime.

What do you think?